keps/0034-instance-health.md

```diff
@@ -0,0 +1,45 @@
+---
+kep-number: 34
+short-desc: Provide a way to monitor readiness of KUDO instance post-deployment
+title: KUDO instance readiness
+authors:
+  - "@alenkacz"
+owners:
+  - "@alenkacz"
+editor: TBD
+creation-date: 2020-09-24
+last-updated: 2020-09-24
+status: provisional
+---
+
+# KUDO instance readiness
+
+## Table of Contents
+
+* [Table of Contents](#table-of-contents)
+* [Summary](#summary)
+* [Motivation](#motivation)
+    * [Goals](#goals)
+    * [Non-Goals](#non-goals)
+
+## Summary
+
+KUDO helps people implement their operators and it's focus is day 2 operations. Part of day 2 is also monitoring your workload readiness after deployment. To help with that, KUDO will expose readiness computed as a heuristic based on readiness of the underlying resources. In the first iteration, readiness will be just a simple heuristic computed from Pods, StatefulSets, Deployments, ReplicaSets, DaemonSets and Services (let's call them *readiness phase 1 resources*).
```

Comment:
the reason I ask about service as a readiness phase 1 resources... is I don't know what it means to know a service is ready?   is there a way to reflect that service is ready?